### PR TITLE
UI Improvement (Level 3)

### LINKED ISSUE #192: Introduction with the epitech logo

### LINKED ISSUE #196: Add level bar for audio Settings

### LINKED ISSUE #198: Hide Mouse Cursor in-game

### LINKED ISSUE #200: Add Epitech Logo in Credit Scene

### LINKED ISSUE #201: Add Retry Button In Score Scene

### LINKED ISSUE #202: The Clock shouldn't start before the intro animation ends

### LINKED ISSUE #213: The Timer chrono should have a correct format

### LINKED ISSUE #218: Fix readme badges

### PR DESCRIPTION
**Pull request by Arthi-chaud**
More Improvements on the UI:

- [x] The Ingame Timer's format has been fixed (Fix #213)
- [x] The Ingame Clock's doesn't start until the camera has settled (Fix #202)
- [x] A retry button added in the score scene (Fix #201)
- [x] Add the EPI logo in the credit scene (Fix #200)
- [x] In The settings, a level bar for audio levels  (Fix #196)
- [x] In Game, the cursor is hidden (Fix #198)
- [x] The Intro Animation include the EPITETECH logo omfg yass (Fix #192)
- [x] A little gift in the credit scene (hushhh)
- [x] Fix Badges on README (Fix  #218)
- [x] Add A-MA-ZING backgrounds because we're worth it

## Information
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Non-breaking changes

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation

### If not needed, remove this section
If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did
and what alternatives you considered, etc...

### CODE REVIEW

**Review by TrueBabyChaise — APPROVED** [review]

**Review by TrueBabyChaise — APPROVED** [review]

**Review by Octopus773 — APPROVED** [review]

**Review by EternalRat — APPROVED** [review]

**Review by GitBluub — APPROVED** [review]

File: sources/Runner/CreditScene.cpp [hunk 1]
```diff
@@ -66,6 +69,24 @@ namespace BBM
 
 				texture->use("assets/buttons/button_back_hovered.png");
 			});
+		scene->addEntity("racoon")
```
[the remark this answers is not in the document]
Comment (AnonymusRaccoon) [reply sources/Runner/CreditScene.cpp:72]: AHHHHHHHHHHHHHHHHH

RACCOON
CC

File: sources/System/IntroAnimation/IntroAnimationSystem.cpp [hunk 2]
```diff
@@ -25,25 +26,47 @@ namespace BBM
...
-            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), BLACK);
+            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), WHITE);
+        static auto &epitechLogo = scene->addEntity("epitech logo")
+			.addComponent<PositionComponent>(1920 / 2 - 150, 1080 / 2 - 61, 0)
+			.addComponent<Drawable2DComponent, RAY::Texture>("assets/epitech.png");
         static short letterCounter = 0;
 
         switch (component.currentStep)
         {
+        case IntroAnimationComponent::animationSteps::epitechLogo:
```
Comment (AnonymusRaccoon) [inline sources/System/IntroAnimation/IntroAnimationSystem.cpp:54]: Is it normal that we don't see an epitech logo while launching the game?

**Review by AnonymusRaccoon — APPROVED** [review]

File: sources/System/IntroAnimation/IntroAnimationSystem.cpp (hunk 2 above)
Comment (Octopus773) [reply sources/System/IntroAnimation/IntroAnimationSystem.cpp:54]: I see it

Comment (AnonymusRaccoon) [reply sources/System/IntroAnimation/IntroAnimationSystem.cpp:54]: I didn't pull. Oups

**Pull request merged**